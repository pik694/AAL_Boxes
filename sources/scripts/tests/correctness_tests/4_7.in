4
34 86 26
68 79 27
38 38 61
30 23 100

