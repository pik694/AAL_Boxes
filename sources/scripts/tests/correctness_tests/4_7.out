3 0 2 1 
378152->378152
